### PR TITLE
[@astrojs/lit] lit 3.0 compatibility

### PR DESCRIPTION
**Pull request by Geoffrey-Pliez**
## Changes
upgrade to lit 3.0

## Testing

Tested with npm command test from package.json

### CODE REVIEW

File: packages/integrations/lit/package.json
Comment (Geoffrey-Pliez) [reply packages/integrations/lit/package.json]: it's done ;)

**Review by lilnasy — CHANGES_REQUESTED** [review]
This is a much requested feature! Thank you for working on this.

I'm going to block this until we merge and release a version of `@astrojs/lit` that continues to work with `lit@^2`.
- https://github.com/withastro/astro/pull/8826
- https://github.com/withastro/astro/pull/8820

Users could then upgrade their projects from lit 2.0 to 3.0 at their own pace.

**Comment by matthewp** [comment]
@lilnasy if no one is working on fix for the old version then I think we should just go ahead and release this.

**Comment by lilnasy** [comment]
The fix landed and was released as the linked PRs show, and I dismissed my review when it did. I am not blocking this.

**Pull request merged**
